pegasus/helpers/form_helpers.rb

```diff
@@ -1,8 +1,13 @@
 require 'digest/md5'
 require lib_dir 'forms/pegasus_form_validation'
+require_relative '../router'
```

Comment:
Should I be concerned about adding this? Note that the tests error with `NameError:         NameError: uninitialized constant DB` without it.